bin/restydoc-index

```diff
@@ -139,13 +139,14 @@ sub wanted {
             or die "cannot copy $docfile to $podfile: $!\n";
 
     } else {
-        my $mdfile = $docfile;
+        my $quoted_mdfile = shell_quote($docfile);
```

Comment:
You should omit the parentheses otherwise the sub prototype check is never performed. You need to declare the sub at the beginning of this file though.